index.html

```diff
@@ -5,104 +5,123 @@
     <meta name="viewport" content="width=device-width, initial-scale=1.0">
     <meta http-equiv="X-UA-Compatible" content="ie=edge">
     <link rel="stylesheet" href="style.css">
-    <link rel="stylesheet" href="mobile.css">
-    <link href="https://fonts.googleapis.com/css?family=Lato:400,700&amp;subset=latin-ext" rel="stylesheet">
-    <title>strona</title>
+    <link rel="stylesheet" href="nav.css">
+    <link rel="stylesheet" href="header.css">
+    <link rel="stylesheet" href="main.css">
+    <link rel="stylesheet" href="section.css">
+    <link rel="stylesheet" href="footer.css">
+    <link rel="stylesheet" href="reset.css">
+    <link href="https://fonts.googleapis.com/css?family=Rubik:400,700i&amp;subset=latin-ext" rel="stylesheet">
+    <title>dokument</title>
 </head>
 
-
 <body>
-    <nav class="logo">
-                <a href="" class="logo1">market</a>           
-                <a href="" class="logo2">buy now</a>
-                <a href="" class="logo3">Remove frame</a>
-    </nav> 
-
-    <nav class="logotwo">
-                <a href="" class="anny">Anny</a>
-                <a href="" class="home">home</a>
-                <a href="" class="link">about</a>
-                <a href="" class="link">portfolio</a>
-                <a href="" class="link">services</a> 
-                <a href="" class="link">contact</a>
+    <nav class="navbar">
```

Comment:
Czy używamy tej klasy gdzieś w CSS? Jeżeli nie najlepiej ją usunąć, żeby zachować porządek w HTML. W innym przypadku w przyszłości kiedy będziemy musieli coś modyfikować możemy się trochę pogubić i nie będziemy wiedzieli czy możemy skasować jakąś klasę albo do czego ona tak naprawdę służy.
PS to nie jedyna klasa/id, która nie jest używana w CSS.